pkg/hostagent/api/api.go

```diff
@@ -1,5 +1,6 @@
 package api
 
 type Info struct {
-	SSHLocalPort int `json:"sshLocalPort,omitempty"`
+	SSHLocalPort int    `json:"sshLocalPort,omitempty"`
+	DriverName   string `json:"driverName,omitempty"`
```

Comment:
- Can we eliminate this from the `Info`?  Just like CPU nums, arch, memory, etc.
- If we really can't remove this, this should be named like `vmType` or `vmDriverName`, as we also have a bunch of other "drivers" such as mount driver, network driver, etc.
- We can also consider marshaling the entire YAML as a JSON here if we are going to copy other YAML properties too here

Reply:
- As drivers are already relying on store, the moment we rely on driver it will have this cyclic import problem. We can look into resolving it as well.
- If am not able to resolve the above issue i will look into moving other properties to rely on this info endpoint itself by YAML to JSON conversion in info endopoint, so that store will simply depend on hostagent info and not assume anything directly from configuration

Reply:
@AkihiroSuda 
One way i could think of solving it is, our pid names can always be equal to vmType. This way, in instance.Inspect() we can simply rely on vmType to get according pid.

But we could also argue that with this assumption we don't even need Name() in driver interface :)

Let me know if this is fine. In that case, i will maybe get rid of Name itself fully from driver and go with vmType itself.

Reply:
@AkihiroSuda 
One way i could think of solving it is, our pid names can always be equal to vmType. This way, in instance.Inspect() we can simply rely on vmType to get according pid.

But we could also argue that with this assumption we don't even need Name() in driver interface :)

Let me know if this is fine. In that case, i will maybe get rid of Name itself fully from driver and go with vmType itself.

Reply:
> go with vmType itself

:+1:

Reply:
Done the changes